### PR TITLE
docs: install cocoapods for mac m1 users with brew

### PR DESCRIPTION
**Pull request by luism3861**
this PR add install cocoapods with brew.

Reference: https://formulae.brew.sh/formula/cocoapods#default

### CODE REVIEW

File: docs/_markdown-m1-cocoapods.mdx [hunk 1]
```diff
@@ -8,4 +8,10 @@ Mac M1 architecture is not directly compatible with Cocoapods. If you encounter
 
 These commands install the `ffi` package, to load dynamically-linked libraries and let you run the `pod install` properly, and runs `pod install` with the proper architecture.
 
+Recently [brew](https://formulae.brew.sh/formula/cocoapods#default) is already supported cocoapods for Mac M1 simply running:
+
+- `brew install cocoapods`
```
Comment (cortinico) [inline docs/_markdown-m1-cocoapods.mdx:13]: @cipolleschi should this just replace the older instructions?

Also how this interacts with cocoapods installed form the Gemfile?
https://github.com/facebook/react-native/blob/a223874cb5c69be1947d43db4d74371c43c90e5d/template/Gemfile#L6

Comment (luism3861) [reply docs/_markdown-m1-cocoapods.mdx:13]: @cortinico any updates about this PR?

Comment (cortinico) [reply docs/_markdown-m1-cocoapods.mdx:13]: Yes sorry. We discussed this internally and we agree on suggesting to use cocoapods via bundler and not install it via brew, as that allows us to align the version and make sure we're recommending only supported ones.

**Comment by cortinico** [comment]
@cipolleschi what's the latest guidance here?

**Pull request closed without merging**